plasma/plasma-mvp/morevp-simplified.md

```diff
@@ -0,0 +1,87 @@
+Plasma MVP requires the use of confirmation signatures to prevent the operator from including a valid transaction time-wise after an invalid transaction.
+Without confirmation signatures, the Plasma MVP exit priority mechanism would process the invalid exit before the valid exit. 
+
+Confirmation signatures are generally bad user experience.
+Users must sign a transaction, wait for it to be included in a block, and sign a second signature attesting that the transaction was included in a valid block.
+Although clients with “hot” wallets can quickly (and usually automatically) sign these secondary signatures, clients with “cold” wallets will face an annoying user-flow (e.g. clicking some button on a hardware wallet).  
+
+Confirmation signatures must also be published to the Plasma chain to avoid certain attack vectors.
+This significantly reduces the amount of block space available for transactions. 
+
+This motivates the search for an alternative exit game that removes the need for confirmation signatures and simultaneously ensures all valid UTXOs can be processed before all invalid UTXOs. 
+
+## More Viable Plasma
+
+More Viable Plasma (“MoreVP”) proposes an exit game that removes the need for confirmation signatures and ensures correct transaction processing order.
+
+### Basic Exit Game
+
+Under this new protocol, we allow owners of both the *inputs or outputs* to any transaction to attempt an exit. 
+
+If the owner of an input `in` to a transaction `tx` wants to exit, the owner must prove that:
+1. `tx` is correctly formed and properly signed.
+2. One of the inputs to `tx` has already been spent in some transaction other than `tx`.
+3. `in` has not been spent in any transaction other than `tx`.
+
+If the owner of an output `out` of a transaction `tx` wants to exit, the owner must prove that:
+1. `tx` is correctly formed and properly signed.
+2. None of the inputs to `tx` have already been spent in some transaction other than `tx`.
+3. `out` has not been spent.
+
+The first requirement in both games ensures that the transaction is correctly formed (input amounts are greater than or equal to outputs amounts, referencing valid inputs) and signatures over the inputs come from their correct respective owners.
+
+The second requirement has something to do with a concept we call transaction “canonicity.”
+Simply defined, a transaction is “canonical” if none of the transaction’s inputs have been previously spent.
+Let's suppose there exist two transactions `tx` and `other_tx` such that `tx` and `other_tx` share at least one input.
+If `tx` *is* included in the Plasma chain, then `tx` is not canonical if `other_tx` is included before `tx` (in an earlier block or in the same block and an earlier transaction index).
+If `tx` is *not* included in the Plasma chain, then `tx` is not canonical if `other_tx` exists.
```

Comment:
"exists" is too vague here, maybe "revealed" or "submitted on-root during the exit game", considering some strict definition of that?

Reply:
Sure.